### PR TITLE
Minor changes to faq

### PR DESCRIPTION
**Pull request by Tabs16**
Hello,
I have made some grammatical changes to FAQ section. Moreover, I have fixed the navbar to the top in order to allow the user to jump to any section easily. 
Please review :)

### CODE REVIEW

**Review by CuriousLearner — CHANGES_REQUESTED** [review]
Hi, 

Thanks for your contribution!

The logo gets stripped off with the fixed header. Can you please fix that?

screen shot 2017-07-04 at 5 57 20 pm

**Comment by Tabs16** [comment]
@CuriousLearner ,done :)

File: css/custom.css [hunk 1]
```diff
@@ -353,3 +353,6 @@ button {
         opacity: 1;
     }
 }
+.col-md-8 {
```
Comment (CuriousLearner) [inline css/custom.css:356]: Not a good way to play with bootstrap grid. Please use classes as needed.

You do not need to add extra style to `col-md-8`.

Also, the logo gets stripped off on small screens:

screen shot 2017-07-07 at 10 56 56 pm

**Review by CuriousLearner — CHANGES_REQUESTED** [review]
Review mentioned inline.

**Comment by theskumar** [comment]
Hi @Tabs16, there was of change to html theme section and the latest is not available at https://pythonindia.github.io/inpycon2017/faqs.html

I've to close this PR, since it can not be merged. If you think there are any issue with the current FAQs would be happy to merge any PRs. 

Thanks again!

**Pull request closed without merging**